pkgs/applications/networking/mumble/default.nix

```diff
@@ -72,66 +50,52 @@ let
     type = "mumble";
 
     nativeBuildInputs = [ qt5.qttools ];
-    buildInputs = [ libopus libsndfile speex qt5.qtsvg rnnoise ]
+    buildInputs = [ libopus libsndfile speex qt5.qtsvg rnnoise flac libogg libvorbis ]
       ++ lib.optional stdenv.isLinux alsa-lib
```

Comment:
`meta.platforms` is set to `platforms.linux`. Do we need this check?

Reply:
Removed the check for `platforms.linux`. Instead, I let it depend on `!jackSupport`